kotlin/src/test/KotlinExample.kt

```diff
@@ -55,7 +54,7 @@ class KotlinExample {
         if(dash.dashboard_elements.isNullOrEmpty()) {
             return null
         }
-        val element = dash.dashboard_elements!!.filter { element -> (element.title?.toLowerCase() ?: "") == title }
```

Comment:
Hah, was this a bug?

Reply:
Probably. Got lucky with it. A compiler warning about the unused variable prompted me to fix it.